src/DateTimePicker/DateTimePicker.tsx

```diff
@@ -89,16 +89,16 @@ export default class DateTimePicker extends React.Component<DateTimePickerProps>
 			this.dateFormatId = this.currentNls![`longDateFormat_${dateFormat}`];
 		}
 
-		if (this.showTimeSelectOnly) {
-			return;
-		}
-
 		if (timeFormat === undefined) {
 			this.timeFormatId = this.currentNls!["time"];
 		} else if (typeof timeFormat === "string") {
 			this.timeFormatId = this.currentNls![`longDateFormat_${timeFormat}`];
 		}
 
+		if (this.showTimeSelectOnly) {
```

Comment:
lift `portalPopupTo` configuration before constructor returns because it shouldn't be affected by other configs